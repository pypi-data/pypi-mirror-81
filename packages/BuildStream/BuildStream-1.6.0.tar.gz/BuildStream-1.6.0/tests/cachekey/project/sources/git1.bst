kind: import
sources:
- kind: git
  url: https://example.com/git/repo.git
  ref: 6ac68af3e80b7b17c23a3c65233043550a7fa685
