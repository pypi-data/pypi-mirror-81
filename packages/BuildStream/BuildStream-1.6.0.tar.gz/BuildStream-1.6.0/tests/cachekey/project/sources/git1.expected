3dafe98cceea0434cf8a0317db110ce8d7715cf55148f7d01db62c87513f22a5